### ISSUE TITLE
[Suggestion] Star Texture

### ISSUE THREAD

**Issue by Xerma** [tsunamayo/Skywanderers#1753]
This may or may not already be on your to-do list but I think the star(s) texture needs updated to fit alongside the newer graphics we got some time ago after that large refactor. However, seeing a similar but more updated texture when up close to the star couldn't hurt - depending on stars and heat damage that is. I also want to include that both Starmade and Space Engineers have more 'shiny' stars.

### The current star texture:
![20200414011653_1](https://user-images.githubusercontent.com/32974671/79193059-7dd52d80-7def-11ea-9a4f-8ef32b6ca00c.jpg)
### What the star texture could be:
![20200414011718_1](https://user-images.githubusercontent.com/32974671/79193101-8fb6d080-7def-11ea-9bfd-49cb186ff45b.jpg)
### Starmade's star:
![20200414012324_1](https://user-images.githubusercontent.com/32974671/79193117-99403880-7def-11ea-87ea-cce01c15b74b.jpg)
### Space Engineer's star:
![20200414012549_1](https://user-images.githubusercontent.com/32974671/79193155-ac530880-7def-11ea-95ab-9a1f959b0656.jpg)

**Comment by tsunamayo** [comment]
Yes that was a 100% on my to-do list, I will go for something a lot simplier and shiny indeed. Cheers

**Comment by Womble-TC** [comment]
Coloured lighting depending on what colour the sun/star is would be cool. So if you had a red star, the ambient light would be red. I think the game used to have it, and it looked amazing. 
![image](https://user-images.githubusercontent.com/29961381/79281598-fea52f80-7eaa-11ea-8925-aa754f38c43a.png)

**Comment by tsunamayo** [comment]
Problem is that it mess up dramatically the coloring obviously, and people are up to a surprise if they start building in these condition. So maybe only when really close to the sun... Like it's hot in there...

**Comment by Xerma** [comment]
When further away though it should be similar to how lighting is near the gas giant but as you get closer to the star it should be more like the picture above. Plus the lights we have in game apply the same issue with making them appear to be a color that they are not and that's not a huge issue as people should be more so blown away at the fact that they finally have a game where colored lights actually give off that color of light

**Comment by Womble-TC** [comment]
If its possible, maybe try out some more dramatic coloured lighting from starts and see what people think about it. If you think its confusing for new players, perhaps make the whatever system the player starts in have normal white lighting.

**Comment by tsunamayo** [comment]
Only a slight tint was extra confusing to me, completely messed up the color balance.

**Comment by Womble-TC** [comment]
That's understandable, but it would really help make other systems feel more interesting and alien if it made your ship look different from the outside. (plus the "big ass light" on the drone could help you see the difference in colour)

**Comment by Aiwii** [comment]
Environmental Lighting effects would really up the immersion and feel of certain areas. I think it would be really cool!

**Comment by ZachZent** [comment]
I would love to see the colored lighting back, though not as intense as it used to be. A bit of blue with blue stars, orange for orange stars, green for green stars really enhances the environment. Also it looks really really cool.

**Comment by Kaiser-Indrasil** [comment]
I would welcome a slight environmental light tinting as present on this screenshot
![exefile_WQyRRANrLI](https://user-images.githubusercontent.com/63756057/79396265-6cb52980-7f73-11ea-8fd9-fcef43fe2307.png)
Notice the warmer shade of dark grey casted by the orange star, and the blueish dark grey light reflected back from the blue nebula.

Here's the same ship against the same star but in its low orbit:
![exefile_9nFFyXpnqP](https://user-images.githubusercontent.com/63756057/79396467-ecdb8f00-7f73-11ea-8ad3-fa362aa17264.png)
As you can see, the shades are much more profound to compensate for the brightness of the star. There is also heavy tinting on the panels facing the star.

The radius of this effect could be linked to a luminosity stat of a star which can be independent from its size, meaning we could have both small but brightly shining stars and supergiants shining dimly.

**Comment by Spectre25** [comment]
I am a huge fan of games that implement a more realistic lighting mechanic and would love to see similar features implemented in Skywanderers, specifically on the note regarding the stars and ambient lighting. 
Some of these games include: Eve Online, Star Citizen, and especially Elite Dangerous...
![Hw7a4T6ciffHmmvmgS9HDd-320-80](https://user-images.githubusercontent.com/63756032/79412487-e0026f80-7f62-11ea-91c3-76329eb8b64c.jpg)
![hqdefault](https://user-images.githubusercontent.com/63756032/79412048-bac13180-7f61-11ea-8074-cfad67a2803e.jpg)
![nnv5ok41hjpl69odnasu](https://user-images.githubusercontent.com/63756032/79412492-e42e8d00-7f62-11ea-8fa9-b0a50ab0ffb7.png)


This video from Elite Dangerous starts off probably a light year or two from the star and ends with the ship passing the star close to 20au away. This is also very close to the distance of Uranus to give you an idea of the scaling and how much ambient lighting is coming from the star vs the background Stars/Nebula's.
https://youtu.be/9BmCUSidhYQ

This video is from Eve Online, it starts at 30au, or about the same distance as Neptune is to the sun, and ends in the neighborhood of a hundred thousand or so miles from the star. The amount of ambient light at the start clearly isn't correct, more closely reflecting the light we receive at 1au and should actually reflect the light similar to near the end of the Elite Dangerous video. The transition from a starburst bloomed out star into a detailed and defined sun is pretty accurate, though it should be drawn out a little longer and be a gradual transition taking place within 0-.5au from the star. 
Assuming of course fancy sci-fi sunglasses are used so we don't go blind at those distances... lol... But seriously though, being that close or closer to the star, never mind the insane intensity of the light, would considerably darken and blot out the surrounding lights and details when facing the star as Kaiser was stating. Even looking away the amount of light being emitted should still blot out most of the background ambiance, especially if its reflecting of say a ship near by.
https://youtu.be/Tio62dr7Nts

This video basically reiterates my Eve videos transition a little better and slower. Looking like it starts around or just outside of Mercury's orbit and zooming in to a similar distance as the Eve Online video.
https://youtu.be/HlscT7zbK98

While not a great example, Space Engineers with the Space Got Real mod does a pretty fine job as far as ambient lighting goes. Maybe a hair to much bloom but still... lol
![20200415191100_1](https://user-images.githubusercontent.com/63756032/79411025-0de5b500-7f5f-11ea-99e8-5642f95848d8.jpg)

Obviously nothing can beat the real deal, but getting close is always a great thing!
![586473main_iss028e005671_full](https://user-images.githubusercontent.com/63756032/79411164-64eb8a00-7f5f-11ea-81c4-b660e2df51c2.jpg)

**Comment by tsunamayo** [comment]
Thanks a lot for the in-depth survey @Spectre25 and the screen @Kaiser-Indrasil .
I want to improve the lighting model at one point, and make it reflect the environment better. Thats on my to-do list but not very high unfortunately! 
To get an advanced model I would need directional ambient (not just a simple value). But
the real tricky part is gonna be the specular cubemap, as the game is full PBR and we have a lot of metal it is mandatory.
